Geometry/CMSCommonData/python/cmsExtendedGeometry2023XML_cfi.py

```diff
@@ -233,7 +234,7 @@
         'Geometry/MuonCommonData/data/v7/gem21.xml',
         'Geometry/MuonCommonData/data/v2/csc.xml',
         'Geometry/MuonCommonData/data/PhaseII/mfshield.xml',
-        'Geometry/MuonCommonData/data/v8/me0.xml',
+        'Geometry/MuonCommonData/data/PhaseII/me0.xml',
```

Comment:
We have to move to the new ME0 version once we understand the reason of the failure
________________________________
From: Ianna Osborne [notifications@github.com]
Sent: 08 April 2016 16:22
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cms-sw/cmssw] HGCal Eras (#13992)


In Geometry/CMSCommonData/python/cmsExtendedGeometry2023XML_cfi.py<https://github.com/cms-sw/cmssw/pull/13992#discussion_r59031861>:

> @@ -233,7 +234,7 @@
>          'Geometry/MuonCommonData/data/v7/gem21.xml',
>          'Geometry/MuonCommonData/data/v2/csc.xml',
>          'Geometry/MuonCommonData/data/PhaseII/mfshield.xml',
> -        'Geometry/MuonCommonData/data/v8/me0.xml',
> +        'Geometry/MuonCommonData/data/PhaseII/me0.xml',


@lgray<https://github.com/lgray> - ok, no need to change the PR. I can see that @bsunanda<https://github.com/bsunanda>'s changes were reverted by @davidlange6<https://github.com/davidlange6>: c5f6536<https://github.com/cms-sw/cmssw/commit/c5f6536ca5a7852f8a0425de9e1168f6c778999a>

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/13992/files/1c0d36b1ecd23c466f676675d848eaf6774eba36#r59031861>